### ISSUE TITLE
Unexpected parse error in file when using class-name-format in .sass-lint.yml

### ISSUE THREAD

**Issue by CuriousCactus** [AtomLinter/linter-sass-lint#222]
- Atom version: 1.35.1
- linter-sass-lint version: 1.9.0
- Global sass-lint version (if used): 1.12.0
- Platform:
  - 10.13.6 (17G5019) OSX
- Expected behaviour: 
Including this in .sass-lint.yml should hide errors related to class-name-format:
```
  class-name-format:
    - 0
```
- Actual behaviour / issue description:

'Error	sass-lint	Unexpected parse error in file	1:1'

Possibly due to https://github.com/sasstools/sass-lint/issues/1154

**Comment by CuriousCactus** [comment]
On reflection I think this was a specific issue with my .sass-lint.yml (I used `class-name-format` twice).
